### ISSUE TITLE
Events do not work properly

### ISSUE THREAD

**Issue by phprouser** [henrichg/PhoneProfilesPlus#67]
I have an event with an additional sensor type BT headset set. The event worked earlier fine, but now it shows BT headset criteria green (as if BT headset would be connected) and runs the event even if I have disabled BT.

**Comment by henrichg** [comment]
Please write me PPP version. 
Only change for this is ability to configure more then one accessoory (radio buttons changed to checkboxes). 
Maybe in this is bug. I will looking on it.

**Comment by henrichg** [comment]
OK, by me, all is OK in PPP. Maybe in your device was not received broadcast for bluetooth headset state change (disconnect).
- BluetoothHeadset.ACTION_AUDIO_STATE_CHANGED
- BluetoothHeadset.ACTION_CONNECTION_STATE_CHANGED
In these bradcasts is extra data for bluetooth state. State is aved into PPP shared preferences and then his called event handler.

Is in your event used Accessories sensor?

**Comment by henrichg** [comment]
OK, if you want, install and test this beta version: https://drive.google.com/file/d/1KMasYeFp47knvOJkmQyvuXbcFPNwyF43/view?usp=drivesdk

In it is small change in Accessory sensor for Bluetooth headsets.

Note: in this version is replaced Google Maps with osmdorid library and Google Play Services Location with Android Location API. PPP is ready for the F-Droid. ;-)

**Comment by phprouser** [comment]
Hi, thanks for your help. I have PPP version 4.2.0.3 and I used Accessories sensor. But it seems the problem was caused by not receiving state change as you indicated in your second response. I have now reconnected and disconnected several times and now this seems to work. So, I assume the problem is caused if the power of the headset is switched off (or battery drains) outside Bluetooth coverage. If this is the reason, then this is quite rare, and now knowing it I can easily fix this if appears again. Thanks again.
